yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -404,7 +417,7 @@ outer:
 	return nil
 }
 
-func (yb *TargetYugabyteDB) InitEventChannelsMetaInfo(migrationUUID uuid.UUID, numChans int, startClean bool) error {
+func (yb *TargetYugabyteDB) InitEventChannelsMetaInfo(migrationUUID uuid.UUID, numChans int, startClean bool, tableNames []string) error {
 	err := yb.connPool.WithConn(func(conn *pgx.Conn) (bool, error) {
 		if startClean {
 			startCleanStmt := fmt.Sprintf("DELETE FROM %s where migration_uuid='%s'", EVENT_CHANNELS_METADATA_TABLE_NAME, migrationUUID)
```

Comment:
Define separate functions for `clearChannelsMetadata()`, `clearEventStatsByTable()`, `initChannelMetadata()`, and `initEventStatsByTableName()`.

Reply:
With `start-clean` we should clean the complete state of `import_data_event_count_by_table` for that `migrationUUID` right?